Fix these:

./tests/test_games_api.py:1:1: F401 'json' imported but unused
./api/leaderboard/views.py:9:1: E302 expected 2 blank lines, found 1

Add a basic test for GET /api/leaderboard/, something like this:

- add a few users to db with different win percentages
- get /api/leaderboard/
- assert 200 ok
- assert users shown in correct order

--- a/api/leaderboard/views.py
+++ b/api/leaderboard/views.py
@@ -6,8 +6,9 @@ from leaderboard.serializers import LeaderSerializer
 
 from core.models import User
 
+
 class LeaderViewSet(ModelViewSet):
-    queryset = User.objects.all().order_by("win_percentage")
+    queryset = User.objects.all().order_by("-win_percentage")
     serializer_class = LeaderSerializer
 
     permission_classes = []

--- a/tests/test_games_api.py
+++ b/tests/test_games_api.py
@@ -1,4 +1,3 @@
-import json
 import pytest
 from rest_framework import status
 from rest_framework.reverse import reverse

--- a/tests/test_leaderboard.py
+++ b/tests/test_leaderboard.py
@@ -1,0 +1,37 @@
+import pytest
+from rest_framework import status
+from rest_framework.reverse import reverse
+from core.models import User
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_leaderboard(authenticate, dummy_user):
+
+    User.objects.create_user(
+        username='l1',
+        password="l11",
+        win_percentage=0.20,
+        wins=20,
+        losses=100
+    )
+    user = User.objects.create_user(
+        username='l3',
+        password="l33",
+        win_percentage=0.30,
+        wins=30,
+        losses=100
+    )
+    # get_user_model().objects.create_user(
+    User.objects.create_user(
+        username='l2',
+        password="l22",
+        win_percentage=0.10,
+        wins=1,
+        losses=10
+    )
+    client = authenticate(dummy_user)
+    response = client.get(reverse('leaderboard:user-list'))
+    users = User.objects.all().order_by("-win_percentage")
+    assert users[0] == user
+    assert response.status_code == status.HTTP_200_OK

